### PR TITLE
Add a section explaining SSL setup

### PR DESCRIPTION
**Pull request by tlunter**
@bfulton @nahiluhmot

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,6 +78,22 @@ irb(main):004:0> Docker.options
 
 Before doing anything else, ensure you have the correct version of the Docker API. To do this, run `Docker.validate_version!`. If your installed version is not supported, a `Docker::Error::VersionError` is raised.
 
+### SSL
```
Comment (nahiluhmot) [inline README.md]: Maybe mention that you usually only need to do this when using boot2docker?

Comment (tlunter) [inline README.md]: Actually, the only reason why I'm adding this is because someone wasn't using boot2docker and didn't know what to do.

Comment (nahiluhmot) [inline README.md]: Ah, :+1:

**Comment by bfulton** [comment]
+1

**Pull request merged**